examples/generate-har.js

```diff
@@ -0,0 +1,17 @@
+// Details at: https://github.com/GoogleChrome/puppeteer/issues/1916
+
+const puppeteer = require('puppeteer');
+const PuppeteerHar = require('puppeteer-har');
```

Comment:
let's just add a reference to `puppeteer-har` under the "testing" section in https://github.com/GoogleChrome/puppeteer/tree/master/examples

Reply:
Oh, and not actually add the new file `examples/generate-har.js`?